docs/integrations/sources/netsuite.md

```diff
@@ -4,8 +4,10 @@ One unified business management suite, encompassing ERP/Financials, CRM and ecom
 
 This connector implements the [SuiteTalk REST Web Services](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/chapter_1540391670.html) and uses REST API to fetch the customers data.
 
+
 ## Prerequisites
 * Oracle NetSuite [account](https://system.netsuite.com/pages/customerlogin.jsp?country=US)
+* [Netsuite Analytics](https://www.netsuite.com/portal/products/analytics.shtml)
```

Comment:
I think we should understand if this is a paid feature and call this out in the copy.

Reply:
It is not a feature that needs to separately paid for. It's part of the subscription cost for Netsuite.